assets/chezmoi.io/docs/developer-guide/architecture.md

```diff
@@ -10,28 +10,27 @@ example:
 go doc -all -u github.com/twpayne/chezmoi/v2/internal/chezmoi
 ```
 
-You can also [browse chezmoi's generated documentation
-online](https://pkg.go.dev/github.com/twpayne/chezmoi/v2).
+You can also [browse chezmoi's generated documentation online][go-docs].
 
 ## Directory structure
 
 The important directories in chezmoi are:
 
-| Directory                        | Contents                                                                                                                                                       |
-| -------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `assets/chezmoi.io/docs/`        | The documentation single source of truth. Help text, examples, and the [chezmoi.io](https://chezmoi.io) website are generated from the files in this directory |
-| `internal/chezmoi/`              | chezmoi's core functionality                                                                                                                                   |
-| `internal/cmd/`                  | Code for the `chezmoi` command                                                                                                                                 |
-| `internal/cmd/testdata/scripts/` | High-level tests of chezmoi's commands using [`testscript`](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript)                                     |
+| Directory                        | Contents                                                                                                                                            |
+| -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `assets/chezmoi.io/docs/`        | The documentation single source of truth. Help text, examples, and the [chezmoi.io][website] website are generated from the files in this directory |
+| `internal/chezmoi/`              | chezmoi's core functionality                                                                                                                        |
+| `internal/cmd/`                  | Code for the `chezmoi` command                                                                                                                      |
+| `internal/cmd/testdata/scripts/` | High-level tests of chezmoi's commands using [`testscript`][testscript]                                                                             |
```

Comment:
The testscript link doesn't render correctly.

Reply:
I’ll look at that later. It should work.